tools/src/generator/templates/class.ejs

```diff
@@ -24,7 +24,7 @@ export interface I<%= className %><%= capitalizeFirstLetter(functionDef.name) %>
    * @type {<%= functionParameter.type %>}
    * @memberof I<%= className %><%= capitalizeFirstLetter(functionDef.name) %>
    */
-  <%= functionParameter.name %><% if(functionParameter.optional){ %>?<% } %>: <%= functionParameter.type %><% if((numberOfParameters - 1) !== index){ %>,<% } %>
+  <%= functionParameter.name %><% if(functionParameter.optional){ %>?<% } %>: <%= functionParameter.type %>;
```

Comment:
Special mention here: In TypeScript, properties of an interface are separated by semicolons, not comma: https://www.typescriptlang.org/docs/handbook/interfaces.html